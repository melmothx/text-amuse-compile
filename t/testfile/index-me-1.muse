#title Test Index 1
#lang hr

<comment>
INDEX testć: Kazalo imena
\testx: {&test}
</comment>

<comment>
INDEX imena: \crash{Žćđ}
Krleža: Krleža, Miroslav
Miroslav Krleža: Krleža, Miroslav
Andrić: Andrić, Ivo
*em*: % test me %&_ without characters with curly because it breaks the test
{^legittest^}: Legit test$
{"legittest"}: Legit test$
\test: another & test
Prova~~prova: Prova
</comment>

<comment>
INDEX mjesta: Kazalo mjesta
Ci
Ča
Ći
Dubrovnik
Džamija
Žu

</comment>


STARTHERE



            * * * * *

{^legittest^}

            * * * * *

Ivo Andrić...

            * * * * *

Krleža

            * * * * *

Miroslav Krleža

            * * * * *

\test

            * * * * *

\test

            * * * * *

{"legittest"}

            * * * * *

Prova~~prova Prova~~prova Ci Dubrovnik {"legittest"}


            * * * * *

*em*  Prova~~prova *em*

            * * * * *

\testx \testx

            * * * * *

*em*

            * * * * *

            * * * * *

Ci Džamija
Dubrovnik

            * * * * *

Ča Ci Ći Dubrovnik
Džamija

            * * * * *

Ći

            * * * * *

Dubrovnik

            * * * * *

Džamija

            * * * * *

Žu

ENDHERE            
