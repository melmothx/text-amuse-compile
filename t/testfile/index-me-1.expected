


\clearpage


\index[imena]{Legit test\$}\{\^{}legittest\^{}\}


\clearpage


Ivo \index[imena]{Andrić, Ivo}Andrić\dots{}


\clearpage


\index[imena]{Krleža, Miroslav}Krleža


\clearpage


\index[imena]{Krleža, Miroslav}Miroslav Krleža


\clearpage


\index[imena]{another \& test}\textbackslash{}test


\clearpage


\index[imena]{another \& test}\textbackslash{}test


\clearpage


\index[imena]{Legit test\$}\{"legittest"\}


\clearpage


\index[imena]{Prova}Prova~prova \index[imena]{Prova}Prova~prova \index[mjesta]{Ci}Ci \index[mjesta]{Dubrovnik}Dubrovnik \index[imena]{Legit test\$}\{"legittest"\}


\clearpage


\index[imena]{\% test me \%\&\_ without characters with curly because it breaks the test}\emph{em}  \index[imena]{Prova}Prova~prova \index[imena]{\% test me \%\&\_ without characters with curly because it breaks the test}\emph{em}


\clearpage


\textbackslash{}testx \textbackslash{}testx


\clearpage


\index[imena]{\% test me \%\&\_ without characters with curly because it breaks the test}\emph{em}


\clearpage


\clearpage


\index[mjesta]{Ci}Ci \index[mjesta]{Džamija}Džamija
\index[mjesta]{Dubrovnik}Dubrovnik


\clearpage


\index[mjesta]{Ča}Ča \index[mjesta]{Ci}Ci \index[mjesta]{Ći}Ći \index[mjesta]{Dubrovnik}Dubrovnik
\index[mjesta]{Džamija}Džamija


\clearpage


\index[mjesta]{Ći}Ći


\clearpage


\index[mjesta]{Dubrovnik}Dubrovnik


\clearpage


\index[mjesta]{Džamija}Džamija


\clearpage


\index[mjesta]{Žu}Žu


